tests/cephfs/cephfs_utilsV1.py

```diff
@@ -245,7 +245,7 @@ def deamon_name(node, service):
             sudo=True,
             cmd=f"systemctl list-units --type=service | grep {service} | awk {{'print $1'}}",
         )
-        service_deamon = out.strip()
+        service_deamon = out.strip().split()[0]
```

Comment:
Hope you have taken care of the functions already using this as this is in utils

Reply:
as we need to return service_daemon alone to caller, this old logic was not working for mon type service. because mon string grep in above cmd list many other irrelevant services, so we had to capture first row element thats mon service name in new logic. 
mgr and mds name can be obtained with new logic too, verified manually.